include/srdfdom/srdf_writer.h

```diff
@@ -37,8 +37,6 @@
 #ifndef _SRDFDOM_SRDF_WRITER_
 #define _SRDFDOM_SRDF_WRITER_
 
-
-#include <boost/shared_ptr.hpp>
```

Comment:
I know this isn't a big deal, but I've always understood best practice is to list *all* includes that are used in a particular file. e.g. list ``#include<vector>`` in every file that uses vectors, even though its super redundant. This is also what roslint and other linters I've used enforces. Of course we don't actually do this always, but I'm surprised to see you purposefully removing a used include - this is essentially a regression of style / best practice